terraform/modules/services/forecast_pvsite/ecs.tf

```diff
@@ -0,0 +1,49 @@
+# define aws ecs task definition
+# needs access to the internet
+
+resource "aws_ecs_task_definition" "ecs-task-definition" {
+  family                   = var.app-name
+  requires_compatibilities = ["FARGATE"]
+  network_mode             = "awsvpc"
+
+  # specific values are needed -
+  # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html
+  cpu    = 1024
+  memory = 7168
```

Comment:
probably can move this down a bit to say 2048 for the moment

Reply:
done in [`d80ca41` (#161)](https://github.com/openclimatefix/ocf-infrastructure/pull/161/commits/d80ca41c108b21698b6384ee8712c6dfeae1bef0)